src/v3/src/components/Widget/index.tsx

```diff
@@ -411,7 +411,9 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
 
   useEffect(() => {
     const asyncEffect = async () => {
-      if (isClientTransaction) {
+      const isEndOfAuthentication = idxTransaction?.status === IdxStatus.TERMINAL
+          && idxTransaction.neededToProceed.length === 0;
```

Comment:
Are there valid cases where the status is terminal but there are still remediations in `neededToProceed`?

Reply:
I performed one more refactor, sorry about this, can you please review one more time 🙏